src/jit/importer.cpp

```diff
@@ -12814,10 +12814,17 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
 #ifdef FEATURE_CORECLR
                     // In coreclr the delegate transparency rule needs to be enforced even if verification is disabled
```

Comment:
> Do we want to delete isDelegateCreationAllowed from JitEE interface?

It would be nice. It should be fine to do it together with the next JITEE interface change.